manifests/g/Geeks3D/FurMark/1.37.2.0/Geeks3D.FurMark.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Geeks3D.FurMark
+PackageVersion: 1.37.2.0
+InstallerType: inno
+Installers:
+- InstallerUrl: https://files1.majorgeeks.com/10afebdbffcd4742c81a3cb0f6ce4092156b4375/benchmark/FurMark_1.37.2.0_Setup.exe
```

```suggestion
- InstallerUrl: https://geeks3d.com/downloads/2023p/FurMark_1.37.2.0_Setup.exe
```

Hi @Dragon1573,

This InstallerUrl comes from the PackageUrl. Should it be used instead?

---

No, this is metioned in issue which will be resolve by this.
The url only can download when you click the download button in browser. It would return a html page via winget

---

@stephengillie

---

@stephengillie NO, this suggestion is unacceptable.

- See https://github.com/microsoft/winget-pkgs/pull/137299#discussion_r1473779991
- See https://github.com/microsoft/winget-pkgs/issues/137134#issuecomment-1918481344
- See https://github.com/microsoft/winget-pkgs/issues/137134#issuecomment-1918931138